pypesto/logging.py

```diff
@@ -31,7 +31,7 @@ def log(name: str = 'pypesto',
         If specified, messages are logged to a file with this name.
     """
     logger = logging.getLogger(name)
-    logger.setLevel(level)
+    logging.basicConfig(level=level)
```

Comment:
it shouldn't be necessary to call basicConfig, as this messes with global debuggers ...

Reply:
well i am not too sure but when i kept it at `logger.setLevel(level)` the level was set to info (so everything should have been okay there) but the info messages still were not displayed...

Reply:
Found the error.